sklearn/cluster/bicluster.py

```diff
@@ -156,6 +169,8 @@ def _svd(self, array, n_components, n_discard):
 
         assert_all_finite(u)
         assert_all_finite(vt)
+        if u.shape[1] == 1 and vt.shape[0] == 1:
```

Comment:
Again, explain please? Can you also add a comment maybe? So their's only one row in X?